app/resources/forms.py

```diff
@@ -40,3 +41,14 @@ class ResourceForm(Form):
         Length(5, 5)
     ])
     submit = SubmitField('Add Resource')
+
+
+class ReviewForm(Form):
+    rating = IntegerField('Rating', validators=[
```

Comment:
Good idea. Assuming that the interface will be changed and improved in the future.